packages/e2e-tests/specs/editor/various/preview.test.js

```diff
@@ -206,6 +206,7 @@ describe( 'Preview with Custom Fields enabled', () => {
 	beforeEach( async () => {
 		await createNewPost();
 		await toggleCustomFieldsOption( true );
+		await page.emulateMediaFeatures( [ { name: 'prefers-reduced-motion', value: 'reduce' } ] );
```

Comment:
Why is it only take effect in specific test?

Reply:
`toggleCustomFieldsOption` called before reloads the page. My thinking was maybe we need this on each page load, but Travis doesn't not agree with me anyway.

I'm not sure yet why this PR is not working as intended.